internal/cli/admincmd/install_client.go

```diff
@@ -483,47 +455,38 @@ func (c *InstallClient) showInstallConfiguration(opts *kubernetes.InstallationOp
 	m.Msg("Configuration...")
 }
 
-func (c *InstallClient) fillInMissingSystemDomain(ctx context.Context, domain *kubernetes.InstallationOption) error {
-	if domain.Value.(string) == "" {
-		ip := ""
-		s := c.ui.Progressf("Waiting for LoadBalancer IP on traefik service.")
-		defer s.Stop()
-		err := helpers.RunToSuccessWithTimeout(
-			func() error {
-				return c.fetchIP(ctx, &ip)
-			}, duration.ToSystemDomain(), duration.PollInterval())
-		if err != nil {
-			if strings.Contains(err.Error(), "Timed out after") {
-				return errors.Wrap(err, deployments.MessageLoadbalancerIP)
-			}
+func (c *InstallClient) waitForTraefikIngressIP(ctx context.Context) error {
```

Comment:
This waiting is probably not needed? https://github.com/epinio/epinio/blob/main/deployments/traefik.go#L161-L163

Reply:
Can this run concurrently to the Traefik deployment ... Nevermind, Traefik is one of those done first, before anything else. ... I would tend to keeping this wait, out of paranoia.

Reply:
I agree, keeping.

Reply:
Fixed up here: https://github.com/epinio/epinio/pull/983/files#diff-34be35943d88eae52a69a908ea43cda8322c3b88cf152da75dd724ead2a333b2R620